internal/api/github/client.go

```diff
@@ -66,13 +65,25 @@ func (c *Client) GetConfiguration(ctx context.Context, ownerLogin, repoName stri
 }
 
 func (c *Client) GetPullRequestReviews(ctx context.Context, ownerLogin, repoName string, prNumber int) ([]*github.PullRequestReview, error) {
-	reviews, nextPage := make([]*github.PullRequestReview, 0, 0), 1
-	for nextPage != 0 {
-		ctxTimeout, fn := context.WithTimeout(ctx, DefaultGitHubOperationTimeout)
-		r, res, err := c.githubClient.PullRequests.ListReviews(ctxTimeout, ownerLogin, repoName, prNumber, &github.ListOptions{
-			Page:    nextPage,
-			PerPage: defaultListOptionsPerPage,
+	reviews := make([]*github.PullRequestReview, 0, 0)
```

Comment:
Previously, the first page requested was 1 (nextPage was set to 1), now it's 0 (default `int` value). Do you know why we requested the 1st page previously?

Reply:
Ah, actually, looking at the pacts I ca see that we originally expected the first page, but it seems like now we don't send a page parameter for the first request. Probably due to the `omitempty` on the `ListOptions` struct.

Reply:
Yep, I mean, we could technically just set it to 1, it would be introducing it in the list options page, just to make it explicit. I would prefer that myself. What do you think?